WTWSMS/common/landed_titles/landed_titles.txt

```diff
@@ -3425,10 +3425,11 @@ e_roman_empire = {
 	
 	culture = roman
 	religion = nicene
+	primary = yes
 	
 	dynasty_title_names = no
 
-	dignity = 100
+	dignity = 500
```

Comment:
That's why it only reads five hundred instead of one hundred thousand. This was a slight boost.